t/t-credentials.sh

```diff
@@ -350,6 +350,9 @@ begin_test "credentials from lfs.url"
   # A 401 indicates URL access mode for the /storage endpoint
   # was used instead of for the lfsapi endpoint
   grep "HTTP: 401" push.log
+  # Ensure we didn't make a second batch request, which means the request
+  # was successfully retried internally
+  [ ! $(grep "tq: retrying object" push.log) ]
```

Comment:
If for some reason this test fails, the error here will be rather unhelpful, since we'll expand the command substitution into multiple words, even potentially inserting newlines.  We could fix this if we wrote this with quotes around the command substitution, like so:

```sh
  [ ! "$(grep "tq: retrying object" push.log)" ]
```

The shell will do the right thing with the nested quote marks.  There are also a few places below that do the same thing.